.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: ci
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: denoland/setup-deno@main
+        with:
+          deno-version: "1.31.1"
```

Comment:
Do we want to pin the version here? Not sure what the standard is accross other Deno repos.

Reply:
hmm I don't have a preference. but let's use 1.x

Reply:
Sounds good to me.

Reply:
Looks like most of the other repos do `1.x` or canary